source/_docs/lockr.md

```diff
@@ -0,0 +1,162 @@
+3. Visit the Lockr Configuration page  (ex. `http://my-site.pantheonsite.io/wp-admin/admin.php?page=lockr-site-config`)
+
+![WP Lockr Configuration example](/source/docs/assets/images/lockr_wpconfig.png)​
+
+4. Enter your email address and Lockr will take care of the rest! (note: For first time users, you will be receiving an email with instruction, if an existing Lockr users should use the email address they used to create their Lockr account. After submitting the request to register the site, you will be prompted for your Lockr password to confirm adding the site to your account.)
+
+![WP Lockr all keys page example](/source/docs/assets/images/lockr_wpallkeys.png)​
+
+Now you’re set! Your WordPress application is now set up to start entering your keys through the Lockr config. Once you have successfully installed the plugin, you can see a list of keys (and edit/update) by going to **All Keys** under the Lockr Menu. Be sure to visit the [Lockr patch library](https://github.com/lockr/lockr-patches/tree/wp) for the latest patches to your favorite plugins.
+
+
+### WP-CLI Commands
+
+The Lockr plugin contains a number of WP-CLI commands to make registering the site, and setting/getting a key simple through the command line.
+
+```nohighlight
+lockr register-site --email=[<Lockr email address>] --password=[<Lockr account password>]
+```
+This command will register the site with Lockr to the email address provided. The password is only necessary for email addresses already with a Lockr account. This is useful for automated deployment from a custom upstream using Quicksilver.
+
+
+```nohighlight
+lockr lockdown
+```
+Run this command and Lockr will go to a [patch library](https://github.com/lockr/lockr-patches/tree/wp) and automatically patch your existing plugins that do not currently integrate natively with Lockr.
+
+
+```nohighlight
+lockr get-key [key name]
+```
+This command will get the key from Lockr and decrypt it. This is a useful command to program in automated functionality in Quicksilver
+
+
+```nohighlight
+lockr set-key --name=[key name] --label=[key label] --value=[key value]
+```
+This command will encrypt a key and send it to Lockr. This is useful during site migrations or automated deployments of new sites through Quicksilver.
+
+
+
+## Drupal Installation
+
+Lockr provides a simple-to-use [plugin](https://www.drupal.org/project/lockr) for Drupal’s [Key Module](https://www.drupal.org/project/key) to manage your site’s keys and secrets. By pairing together Lockr, [Encrypt](https://www.drupal.org/project/encrypt) and the Key module Drupal can finally take advantage of best security practices of separating the keys away from the data they protect. Lockr is currently available for Drupal 7 and Drupal 8. Visit our [Github page](https://github.com/lockr/lockr-patches/tree/drupal7) for a list of plugins that can be automatically patched.
```

Comment:
Pairing together the [Lockr](https://www.drupal.org/project/lockr), [Encrypt](https://www.drupal.org/project/encrypt), and [Key](https://www.drupal.org/project/key) modules allow Drupal to take advantage of best security practices of separating the keys away from the data they protect. 

Lockr is currently available for Drupal 7 and Drupal 8. See the project's [Github page](https://github.com/lockr/lockr-patches/tree/drupal7) for a list of modules that can be automatically patched.